index.html

```diff
@@ -173,126 +173,364 @@ <h2>
+            is hidden.</p>
+          <p>
+            Once in the <a>Pointer Lock State</a> the <a>user agent</a> has a
+            <a>pointer-lock target</a> and a set of <a>PointerLockOptions</a>. The
+            <a>pointer-lock target</a> must receive all relevant user generated
+            {{MouseEvent}} events (for example: `mousemove`, `mousedown`, `mouseup`, 
+            `click`, and `wheel`) [[ui-events]]. No other elements may receive these
+            events while in <a>Pointer Lock State</a>. Events that require the concept of
+            a mouse cursor must not be dispatched (for example: `mouseover`, `mouseout`,
+            `drag`, and `drop`).
+          </p>
+          <p>
+            <dl data-dfn-for="PointerLockOptions" data-link-for=
+            "PointerLockOptions">
+              While in the <a>Pointer Lock State</a> if the <a>PointerLockOptions</a>'
+              <a>unadjustedMovement</a> member is <code>true</code> the event 
+              coordinates must not be affected by the
+              underlying platform behaviors such as mouse acceleration. In other
+              words, UA must use the APIs provided by the underlying platform to
+              guarantee getting the raw events. If the <a>PointerLockOptions</a>' <a>unadjustedMovement</a> member is
+              <code>false</code> UA must rely on the default behavior of the
+              underlying platform regarding the mouse acceleration.
+            </dl>
+          </p>
+          <p>
+            In the <a>Pointer Lock State</a> the system mouse cursor must be hidden.
+            Movement and button presses of the mouse must not cause the window
+            to lose focus.
+          </p>
+          <p class="note">
+            Synthetic mouse events created by application script act the same
+            regardless of lock state.
+          </p>
+        </p>
+      </section>
+      <section>
+        <h3>
+          `PointerLockOptions` dictionary
+        </h3>
+        <pre class="idl">
+          dictionary PointerLockOptions {
+            boolean unadjustedMovement = false;
+          };
+        </pre>
+        <dl data-dfn-for="PointerLockOptions" data-link-for=
+        "PointerLockOptions">
+          <dt>
+            <dfn>PointerLockOptions</dfn> dictionary
+          </dt>
+          <dd>
+            <p>
+              The options dictionary to customize how the pointer behaves in
+              the locked mode.
+            </p>
+          </dd>
+          <dt>
+            <dfn>unadjustedMovement</dfn> member
+          </dt>
+          <dd>
+            <p>
+              If this value is set to `true`, then the pointer movements will
+              not be affected by the underlying platform modications such as
+              mouse accelaration.
+            </p>
+          </dd>
+        </dl>
+      </section>
+      <section>
+        <h3>
+          `"pointerlockchange"` and `"pointerlockerror"` Events
```

Comment:
Nit: traditionally, for some reason, we don't include the quotes when talking about event names. I agree quotes make more sense, but for consistency it'd be better to remove them.